LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
```

Comment:
![image](https://user-images.githubusercontent.com/94551484/219780282-d4ed4e31-cd41-4008-b064-a4f4ab35a637.png)
- [ ] Kindly wrap the header of your web page including the nav in a `header` element. This is best for HTML semantics.
- [ ] Kindly wrap your logo in an `a` tag with href pointing to the index.html. This is so that when user clicks on the logo, they get taken to the homepage of your site for a better user experience.
- [ ] Kindly wrap the hamburger icon in a button element =, this is because the hamburger is a button that displays the mobile menu when clicked so it is best to wrap it in a button or an a tag

Reply:
I fixed that.